client/app/hosting/translations/Messages_fr_FR.json

```
@@ -270,6 +270,24 @@
   "hosting_tab_CRON_table_language_php7_1": "PHP 7.1",
   "hosting_tab_CRON_table_language_PHP_7_2": "PHP 7.2",
   "hosting_tab_CRON_table_language_php7_2": "PHP 7.2",
+  "hosting_tab_CRON_table_language_PHP_7_3": "PHP 7.3",
+  "hosting_tab_CRON_table_language_php7_3": "PHP 7.3",
+  "hosting_tab_CRON_table_language_ruby2_4": "Ruby 2.4",
+  "hosting_tab_CRON_table_language_RUBY2_4": "Ruby 2.4",
+  "hosting_tab_CRON_table_language_ruby2_5": "Ruby 2.5",
+  "hosting_tab_CRON_table_language_RUBY2_5": "Ruby 2.5",
+  "hosting_tab_CRON_table_language_ruby2_6": "Ruby 2.6",
+  "hosting_tab_CRON_table_language_RUBY2_6": "Ruby 2.6",
+  "hosting_tab_CRON_table_language_python2": "Python 2",
+  "hosting_tab_CRON_table_language_PYTHON2": "Python 2",
+  "hosting_tab_CRON_table_language_python3": "Python 3",
+  "hosting_tab_CRON_table_language_PYTHON3": "Python 3",
+  "hosting_tab_CRON_table_language_node8": "Node 8",
```

It shouldn't be this instead:

```suggestion
  "hosting_tab_CRON_table_language_node8": "Node.js 8",
```

?

---

We should probably think of another way to display those instead of having to add translations each time, can be handled in another PR (Ticket reference created for this: MANAGER-2973)

---

I'd just change Node by Node.js
For the improvment of translation system, i let the works for frontend teams ;)